test/server/cards/02_SHD/upgrades/MandalorianArmor.spec.ts

```diff
@@ -0,0 +1,37 @@
+describe('Mandalorian Armor', function() {
+    integration(function(contextRef) {
+        describe('Mandalorian Armor\'s attached ability', function() {
+            it('If attached unit is a Mandalorian, give a Shield token to it, or dont', function () {
```

Comment:
These two sentences will be concatenated, so they should say something like `Mandalorian Armor\'s attached ability should give a shield to the attached card if it is a Mandalorian unit`. Also you can remove the `describe` since there's only one `it` case and just put the whole name inside of the `it`